convention-develocity-gradle-plugin/README.md

```diff
@@ -15,7 +46,8 @@ This project is structured as follows:
     * `gradle_3_and_earlier` - Applies the convention plugin on a Gradle 3 and earlier (only the Develocity plugin is applied)
     * `gradle_4` - Applies the convention plugin on a Gradle 4 build (the Develocity and CCUD plugins are applied)
     * `gradle_5` - Applies the convention plugin on a Gradle 5 build (the Develocity and CCUD plugins are applied)
-    * `gradle_6_and_later` - Applies the convention plugin on a Gradle 6 and later build (the Develocity and CCUD plugins are applied)
+    * `gradle_6` - Applies the convention plugin on a Gradle 6 build (the Develocity and CCUD plugins are applied)
+    * `gradle_6.9_and_later` - Applies the convention plugin on a Gradle 6.9 and later build (the Develocity and CCUD plugins are applied)
```

Comment:
how come we have gradle_6 and gradle_6_9 folders?

Reply:
I discussed this in a call with @etiennestuder. For other reviewers, Gradle 6.9 is the first version where it is possible to use dynamic versions in plugin blocks. In earlier versions, it is only possible to use a fixed version.